### PR TITLE
feat: create the create the view about admin dashboard, based on the …

### PR DESCRIPTION
**Pull request by blertaazemi**
…pdf "admin home page"

### CODE REVIEW

File: src/OnlineVotingS.API/Controllers/AdminController.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace OnlineVotingS.API.Controllers
+{
+    public class AdminController : Controller
+    {
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/AdminController.cs]: remove this line

File: src/OnlineVotingS.API/Controllers/AdminController.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+namespace OnlineVotingS.API.Controllers
+{
+    public class AdminController : Controller
+    {
+       
+        public IActionResult Dashboard()
+        {
+            return View();
+        }
+        public IActionResult voter()
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/AdminController.cs]: add new line here

Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/AdminController.cs]: Make it with uppercase Voter

File: src/OnlineVotingS.API/Controllers/AdminController.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace OnlineVotingS.API.Controllers
+{
+    public class AdminController : Controller
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/AdminController.cs]: try to use the ControllerBase

Comment (blertaazemi) [reply src/OnlineVotingS.API/Controllers/AdminController.cs]: Nuk po ja kom iden qysh me bo

Comment (adhurimxhemaili) [reply src/OnlineVotingS.API/Controllers/AdminController.cs]: AdminController : ControllerBase

**Pull request merged**